dists/go-ipfs/Makefile

```diff
@@ -7,13 +7,14 @@ dist: versions
 	../../build-go.sh go-ipfs github.com/ipfs/go-ipfs/cmd/ipfs versions 
 	cp versions $(releases)/versions
 
-versions: filtered_versions
-	git ls-remote -t --refs $(repo) | egrep -o "refs/tags/v(.*)" | sed 's/refs\/tags\///' | sort > tag_versions
+versions: FORCE
```

Comment:
This wasn't being re-generated because `versions` was newer than `filtered_versions`.